app/views/steps/shared/_existing_names.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= c.hidden_field :id %>
```

Comment:
What about using a longer format for `c` and `f` variables in these files?

When just looking at these files it won't be that obvious that they refer to `child` and `field` from the loop in the other view.

Reply:
Oh, yes, I just took what was already there (I have to admit in rails app is very common for this kind of form elements to be named just `f` or single letters) but out of the context makes it a bit obscure.

Will rename it and push.

Reply:
Yeah I see if often and I think it's okay inside a loop like it was before when you can see the full reference nearby but here it feels like it's lost its descriptiveness.